CBLClient/Document.py

```diff
@@ -220,3 +220,10 @@ def setData(self, document, data):
         args.setMemoryPointer("document", document)
         args.setDictionary("data", data)
         return self._client.invokeMethod("document_setData", args)
+
+    def setValue(self, document, key, value):
```

Comment:
is getValue method already there?

Reply:
getValue API already exisited in all 3 platforms, but missing on python file. I added missing method